### ISSUE TITLE
Cannot get Host Templates

### ISSUE THREAD

**Issue by steaksauce-** [Icinga/icingaweb2-module-director#2076]
## Expected Behavior

When using director, I should be able to select a host template and view it in the new pane that opens up. I _think_ this happens with other templates, and isn't limited to the error below, meaning that after the generateHeader function, there's another function that bails out.

## Current Behavior

Currently, when doing this, I get the following error:
```
Uncaught Error: Call to undefined method Icinga\Module\Director\Web\Table\HostTemplateUsageTable::generateHeader() in /usr/share/icingaweb2/modules/director/library/Director/Web/Table/TemplateUsageTable.php:101
Stack trace:
#0 /usr/share/icingaweb2/modules/director/library/Director/Web/Table/TemplateUsageTable.php(45): Icinga\Module\Director\Web\Table\TemplateUsageTable->__construct(Object(Icinga\Module\Director\Objects\IcingaHost))
#1 /usr/share/icingaweb2/modules/director/library/Director/Web/Controller/TemplateController.php(190): Icinga\Module\Director\Web\Table\TemplateUsageTable::forTemplate(Object(Icinga\Module\Director\Objects\IcingaHost))
#2 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Director\Web\Controller\TemplateController->usageAction()
#3 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('usageAction')
#4 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch( (TemplateUsageTable.php:101)

#0 [internal function]: Icinga\Application\Web->Icinga\Application\{closure}()
#1 {main}
```

## Possible Solution

It seems like the generateHeader() function exists in the ipl/Html/Table area, and that is defined in the header of the file, but for some reason isn't being used. I don't know enough about PHP to exactly know why it's not working.

## Steps to Reproduce (for bugs)

In Icingaweb2 navigate to Director > Hosts > Templates and click on a Host Template.

## Your Environment

* Director version (System - About):

Name | Version
-- | --
setup | 2.7.3
audit | 1.0.1
businessprocess | 2.2.0
cube | 1.1.0
director | master
doc | 2.7.3
fileshipper | 1.1.0
grafana | 1.3.6
incubator | v0.3.0
ipl | v0.4.0
monitoring | 2.7.3
pdfexport | 0.9.1
reactbundle | v0.6.0
reporting | 0.9.2
x509 | 1.0.0

* Icinga Web 2 version and modules (System - About):
2.7.3 (See above for all modules and versions)
* Icinga 2 version (`icinga2 --version`):

* Operating System and version:
Centos 7.7.1908
* Webserver, PHP versions:
`PHP 7.1.33`
`httpd-2.4.6-90`

**Comment by steaksauce-** [comment]
I've upgraded `reactbundle` to 0.7.0 and `incubator` to 0.5.0 after noticing that these were out of date (I was using the default version on the installation instructions for these). After doing this the issue still persists.

**Comment by Thomas-Gelf** [comment]
Looks like you're running a pretty old master version of the Icinga Director. Please either upgrade to the latest stable version or to the latest master. The error you're facing has been fixed a few months ago with de2ad48

**Comment by steaksauce-** [comment]
Thanks -- not sure why I had such an old commit (I pulled the code last week).
`commit 686f168ee339e679dad778cab18f474d84bb3912` was what I pulled using v1.7.1 as per the installation instructions.

Switching to master and then `git pull` fixes my issue at least on this machine. I will try to reproduce via the installation instructions on another machine, but I'll close this for now.
